uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/AbsencePeriod.php

```diff
@@ -258,6 +258,25 @@ public static function getMostRecentStartDateAvailable()
   }
 
   /**
+   * Returns the current AbsencePeriod. That is, the period that contains the
+   * current date. If no such period is found, null will be returned.
+   *
+   * @return \CRM_HRLeaveAndAbsences_BAO_AbsencePeriod|null
+   */
+  public static function getCurrentPeriod()
+  {
+    $period = new self();
+    $period->whereAdd("start_date <= NOW()");
```

Comment:
correct me if I am wrong but as far as I remmber the absence period use date not datetime .. so its probably better to use CURDATE() here